presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -731,8 +731,9 @@ exists as we've rolled back to the previous state.
     -----------+------+-----------+---------
     (0 rows)
 
-Iceberg Connector Limitations
------------------------------
+Table with delete files
+-----------------------
 
-* The ``SELECT`` operations on Iceberg Tables with format version 2 do not read the delete files
-  and remove the deleted rows as of now (:issue:`20492`).
+Iceberg V2 tables support `row-level deletion <https://iceberg.apache.org/spec/#delete-formats>`,
+and Presto supports reading these deleted files, including Position Delete Files and Equality Delete Files.
+When reading, Presto will merge these deleted files to read the latest results.
```

Comment:
```suggestion
When reading, Presto merges these deleted files to read the latest results.
```
Revising sentence to present tense. For references, see the Google developer documentation style guide about [present tense](https://developers.google.com/style/tense).